client/webpack.config.js

```diff
@@ -61,6 +61,7 @@ const config = {
     alias: {
       lib: path.resolve('./app/lib'),
       api: path.resolve('./app/api'),
+      surveyApp: path.resolve('./app/bundles/course/survey'),
```

Comment:
I'm not sure why `surveyApp` is used here, because there is no naming with App now. Would prefer other options include just `survey`, or provide an alias to `./app/bundles/course`, then do something like `import SectionFormDialogue from 'course/survey/containers/SectionFormDialogue'`.

Thoughts?